delphi-checks/src/test/java/au/com/integradev/delphi/checks/FullyQualifiedImportCheckTest.java

```diff
@@ -82,6 +82,18 @@ void testUnresolvedImportShouldNotAddIssue() {
         .verifyNoIssues();
   }
 
+  @Test
+  void testUnitAliasImportShouldNotAddIssue() {
+    var testFile = new DelphiTestUnitBuilder().appendDecl("uses").appendDecl("  AliasName;");
+
+    CheckVerifier.newVerifier()
+        .withCheck(new FullyQualifiedImportCheck())
+        .withSearchPathUnit(new DelphiTestUnitBuilder().unitName("Scope.RealName"))
+        .withUnitAlias("AliasName", "Scope.RealName")
```

Comment:
I think this test case could be a bit stronger. The problem I see is that the test will continue to pass if the unit alias isn't working at all; the import will be unresolved and no issue will be raised.

You could *also* add the unit scope name of `Scope` alongside a new unit `Scope.AliasName`, which will cause an issue to be raised if the alias is not working (but the Check *is*).

```suggestion
        .withSearchPathUnit(new DelphiTestUnitBuilder().unitName("Scope.RealName"))
        .withSearchPathUnit(new DelphiTestUnitBuilder().unitName("Scope.AliasName"))
        .withUnitScopeName("Scope")
        .withUnitAlias("AliasName", "Scope.RealName")
```